<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>711</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,10">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>书名:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameBook"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>出版社:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="namePublisher"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>类别:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameCategory"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>作者:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameAuthor"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>价格:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="price1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>年份:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxYear"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>——</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="price2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,10">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>——</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QTableView" name="view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
